app/public/css/styles.css

```diff
@@ -82,8 +115,10 @@ p {
 path {
   fill: currentColor;
 }
+
+
 /* BUTTON */
-button, .button-like {
```

Comment:
button-like class appears still in styles.css despite I can't find it in the code. Are you sure this change won't mess up the styles. Moreover, btn as an element probably does not exist, you probably wanted to put the class ".btn", but this is imho too generic. Was there any good reason to replace "button" selector with "btn"?